Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM golang:latest
+
+RUN mkdir -p /go/src/github.com/kris-nova/kubicorn/
+
+WORKDIR /go/src/github.com/kris-nova/kubicorn/
+
+COPY apis		apis
```

Comment:
I'm not really sure why we have bunch of that `COPY` commands? Wouldn't `go get` do the needed job?

Reply:
Yes @xmudrii that's true Let me try that, it should work.